webhooks/clusterclass.go

```diff
@@ -123,8 +140,126 @@ func (webhook *ClusterClass) validate(old, new *clusterv1.ClusterClass) error {
 	// Ensure spec changes are compatible.
 	allErrs = append(allErrs, check.ClusterClassesAreCompatible(old, new)...)
 
+	// Ensure no MachineDeploymentClass currently in use has been removed from the ClusterClass.
+	allErrs = append(allErrs, webhook.validateLivingMachineDeploymentClassesNotRemoved(old, new)...)
```

Comment:
```suggestion
	allErrs = append(allErrs, webhook.validateInUseMachineDeploymentClassesNotRemoved(old, new)...)
```
Maybe, living sounds strange to me

Reply:
I think 
```suggestion
	allErrs = append(allErrs, webhook.validateRemovedMachineDeploymentClassesAreNotUsed(old, new)...)
```
Works well in context of the other checks WDYT?